dashboard/db/migrate/20210715223451_create_reviewable_projects.rb

```diff
@@ -0,0 +1,21 @@
+class CreateReviewableProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviewable_projects do |t|
+      t.integer :storage_app_id, null: false
+      t.integer :user_id, null: false
+      t.integer :level_id, null: false
```

Comment:
Others might disagree, but I might enforce user_id and level_id presence at the model level vs. the database level, [something like this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/code_review_comment.rb#L28) (although you don't have to worry about putting in model validations in this PR, just the DB table itself). Doesn't seem 100% fundamental to the feature and would makes it easier to change things in the future if we decide we want projects to be reviewable outside of the context of levels/scripts.

Reply:
I think we should allow level_id to be nullable, not sure if it would make sense to have a project without a user_id though.

Reply:
Ahh I meant level_id and script_id, updated now! Agree re: user_id